### ISSUE TITLE
Configure isort to include source paths

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse#337]
## Description

It would be nice to have our `isort` configurations in our `.pre-commit-config.yaml` file set up to recognize the repo-specific source paths: https://pycqa.github.io/isort/docs/configuration/options.html#src-paths

This would separate internal imports (1st party) from our other dependencies (3rd party) in the import statements. Presently the two are conflated and grouped in the same block:

https://github.com/WordPress/openverse-catalog/blob/dad3cb49618f38aadb3ae7772f98ca77180bf35c/openverse_catalog/dags/database/image_expiration_workflow.py#L8-L11

I believe this is reliant on #334 since we would also need use the repo-specific folders (i.e. `openverse_catalog` for the catalog repo, `api/catalog` and `ingestion-server/ingestion_server` for the API repo).

**Comment by dhruvkb** [comment]
The API already has an [isort config file `.isort.cfg`](https://github.com/WordPress/openverse-api/blob/main/.isort.cfg) so the catalog can define one in the root too. This need not be blocked on #334.

**Comment by AetherUnbound** [comment]
Oh! It even has `known_first_party` defined!

https://github.com/WordPress/openverse-api/blob/55f55de18766854c63c78a39006a5d516744a811/.isort.cfg#L6

Thanks for pointing that out Dhruv 😄

**Comment by dhruvkb** [comment]
isort also merges args with the `.isort.cfg` file so we can put the common parts of the config in the `pre-commit` args and leave the different parts in the config file. Since #334 is merged, this is ready for work!
